libc/src/complex/generic/conjf16.cpp

```diff
@@ -0,0 +1,25 @@
+//===-- Implementation of conjf16 function --------------------------------===//
+//
+// Part of the LLVM Project, under the Apache License v2.0 with LLVM Exceptions.
+// See https://llvm.org/LICENSE.txt for license information.
+// SPDX-License-Identifier: Apache-2.0 WITH LLVM-exception
+//
+//===----------------------------------------------------------------------===//
+
+#include "src/complex/conjf16.h"
+#if defined(LIBC_TYPES_HAS_CFLOAT16)
+
+#include "src/__support/CPP/bit.h"
+#include "src/__support/common.h"
+#include "src/__support/complex_type.h"
+
+namespace LIBC_NAMESPACE_DECL {
+
+LLVM_LIBC_FUNCTION(cfloat16, conjf16, (cfloat16 x)) {
+  Complex<float16> x_c = cpp::bit_cast<Complex<float16>>(x);
+  return (x_c.real - x_c.imag * (cfloat16)1.0i);
```

Comment:
I don't think we should use complex arithmetic here.  Probably something like:
```
x_c.imag = -x_c.imag;
return cpp::bit_cast<cfloat16>(x_c);
```
Also this will be used quite a lot.  Feel free to add it to some support template.

Reply:
sure. added a support template for conjugate and some new tests.